node/src/accountant/mod.rs

```diff
@@ -4394,102 +4032,62 @@ pub mod tests {
     }
 
     #[test]
-    fn separate_transactions_if_still_pending_works() {
-        let statuses = vec![
-            PendingTransactionStatus::Confirmed(make_payment_backup()),
-            PendingTransactionStatus::StillPending(make_transaction_id()),
-            PendingTransactionStatus::Failure(make_transaction_id()),
-        ];
-
-        let (pending, non_pending) = Accountant::separate_transactions_if_still_pending(statuses);
-
-        assert_eq!(
-            pending,
-            vec![PendingTransactionStatus::StillPending(make_transaction_id())]
-        );
-        assert_eq!(
-            non_pending,
-            vec![
-                PendingTransactionStatus::Confirmed(make_payment_backup()),
-                PendingTransactionStatus::Failure(make_transaction_id())
-            ]
-        )
-    }
-
-    #[test]
-    fn update_backup_of_pending_transaction_happy_path() {
+    fn update_pending_payable_fingerprint_happy_path() {
```

Comment:
This test used to involve two transactions; now it involves only one. Is that a problem? Investigate.